docs/products/kafka/howto/terraform-governance-approvals.md

```diff
@@ -0,0 +1,240 @@
+---
+title: Manage approvals in Aiven for Apache Kafka® Governance using Terraform & GitHub
+sidebar_label: Governance with Terraform
+---
+
+import RelatedPages from "@site/src/components/RelatedPages";
+import TerraformSample from '@site/src/components/CodeSamples/TerraformSample';
+
+Aiven for Apache Kafka® Governance lets you manage approval workflows for Apache Kafka topic changes using Terraform and GitHub Actions.
+
+To manage approvals in the Aiven Console, see
+[Manage approvals in the Aiven Console](/docs/products/kafka/howto/approvals).
+
+
+## Prerequisites
+
+- [Authentication token](/docs/platform/howto/create_authentication_token) for an
```

Comment:
Small fix to the link - tokens for application users are created in a different place than the personal tokens. I also suggested changing this to "tokens" since we have moved to naming these "personal tokens" for users and "application tokens" for application users. You could also use "application token" here.

```suggestion
- [Token](/docs/platform/howto/manage-application-users) for an
```